webapp/scripts/functions.js

```diff
@@ -238,6 +266,23 @@ module.exports = {
     return list
   },
 
+  /**
+   * get all files of a file type in a directory 
+   * @param {string} dir directory to search in
+   * @param {array} files array of filenames 
+   * @returns {array} array of filenames
+   */
+  getAllFile(dir, files, extension) {
+    fs.readdirSync(dir).forEach(function (file) {
+      if (fs.statSync(dir + "/" + file).isDirectory()) {
+        files = module.exports.getAllFile(dir + "/" + file, files, extension)
+      } else if (file.slice(file.lastIndexOf('.') + 1) === extension) {
+        files.push(path.join(dir, "/", file))
```

Comment:
You can omit `"/"` in `path.join`. Why don't you use `path.join` in the previous statements as well?